.github/docker/linux_amd64/Dockerfile

```
@@ -0,0 +1,126 @@
+      libc++-${MAJOR}-dev \
+      libc++abi-${MAJOR}-dev \
+      lld-${MAJOR} && \
+    # register alternatives so `clang` points at clang-${MAJOR}, etc.
+    update-alternatives --install /usr/bin/clang      clang      /usr/bin/clang-${MAJOR}       100 && \
+    update-alternatives --install /usr/bin/clang++    clang++    /usr/bin/clang++-${MAJOR}     100 && \
+    update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-${MAJOR} 100
+
+ENV PATH="/usr/lib/llvm-${MAJOR}/bin:${PATH}"
+
+# Install cmake 3.31
+RUN mkdir /cmake_3_31 && \
```

Hadolint (2.12.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)